enterprise/management/commands/backfill_role_assignments.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command for assigning enterprise_learner roles
+to existing linked enterprise users that are missing them.
+"""
+
+import logging
+from time import sleep
+
+from django.contrib import auth
+from django.core.management.base import BaseCommand
+
+from enterprise.constants import ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCustomerUser, SystemWideEnterpriseRole, SystemWideEnterpriseUserRoleAssignment
+from enterprise.utils import batch
+
+log = logging.getLogger(__name__)
+User = auth.get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Management command for assigning enterprise_learner roles to existing enterprise users.
+
+    Example usage:
+        $ ./manage.py backfill_role_assignments
```

Comment:
nit: maybe rename the command to `backfill_learner_role_assignments` just to be more explicit?